dags/multipod/legacy.py

```diff
@@ -133,3 +133,27 @@
       docker_image=DOCKER_IMAGE[SetupMode.STABLE].value,
       test_owner=test_owner.PRIYANKA_G,
   ).run()
+
+  # v4-16 1 slice, v4-8 1 and 2 slices shardings.py test
+  for cores in [8, 16]:
+    if cores == 8:
+      cluster_name = ClusterName.V4_8_MULTISLICE_CLUSTER.value
+    elif cores == 16:
+      cluster_name = ClusterName.V4_16_MULTISLICE_CLUSTER.value
+    for n_slice in [1, 2]:
+      if cores == 16 and n_slice == 2:  # Skip test for 2 slice v4-16
+        break
+      gke_config.get_gke_config(
+          tpu_version=TpuVersion.V4,
+          tpu_cores=cores,
+          num_slices=n_slice,
+          tpu_zone=Zone.US_CENTRAL2_B.value,
+          time_out_in_min=60,
+          test_name=f"maxtext-shardings-v4-{cores}-{n_slice}-slices-{SetupMode.STABLE.value}",
```

Comment:
The test will format in the accelerator and slice count, so this can just be `maxtext-shardings-{SetupMode.STABLE.value}`

Reply:
Makes sense, should I also make the change to other test names `maxtext-perf` and `maxtext-checkpoint-reshard` in this PR?

Reply:
Oh haha I didn't catch that in the other PRs, please do!